examples/bridge/animation-controls/date-format.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Airship bridge</title>
+
+  <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.11/lodash.min.js"></script>
+  <script src="https://libs.cartocdn.com/carto-vl/v1.3/carto-vl.min.js"></script>
+
+  <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.52.0/mapbox-gl.css" rel="stylesheet" />
+  <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v0.52.0/mapbox-gl.js"></script>
+
+  <link rel="stylesheet" href="../../../packages/styles/dist/airship.css">
+  <script type="module" src="../../../packages/components/dist/airship/airship.esm.js"></script>
+  <script nomodule="" src="../../../packages/components/dist/airship/airship.js"></script>
+  <script src="../../../packages/bridge/dist/asbridge.js"></script>
+
+  <style>
+    .as-map-footer {
+      max-width: 812px;
+    }
+  </style>
+</head>
+
+<body class="as-app-body">
+  <div class="as-app">
+    <div class="as-content">
+      <main class="as-main">
+        <div class="as-map-area">
+          <div id="map"></div>
+        </div>
+        <div class="as-map-footer">
+          <div class="as-box">
+            <as-animation-controls-widget animated heading='Animation' description='Try seeking and pausing'>
+            </as-animation-controls-widget>
+          </div>
+        </div>
+      </main>
+    </div>
+  </div>
+  <script>
+    const map = new mapboxgl.Map({
+      container: 'map',
+      style: carto.basemaps.darkmatter,
+      center: [-74.0032059, 40.7482066],
+      zoom: 12,
+    });
+
+    carto.setDefaultAuth({
+      apiKey: 'default_public',
+      username: 'cartovl'
+    });
+
+    const s = carto.expressions;
+    const source = new carto.source.Dataset('taxi_50k');
+
+    const viz = new carto.Viz(`
+      @hour: clusterTime($tpep_pickup_datetime, 'hour', 'America/New_York')
+
+      width: 10
+      strokeWidth: 0
+      color: opacity(rgb(255,255,0),0.5)
+
+      @animation: animation(@hour, 130, fade(0.0,0.0))
+      filter: @animation
+
+      @list: viewportFeatures(@hour)
+      @maxhour: globalMax(@hour)
+      @minhour: globalMin(@hour)
+    `);
+
+    const layer = new carto.Layer('layer', source, viz);
+    layer.addTo(map);
+
+    const timeWidget = document.querySelector('as-animation-controls-widget');
+
+    const bridge = new AsBridge.VLBridge({
+      carto: carto,
+      map: map,
+      layer: layer,
+      source: source
+    });
+
+    bridge.animationControls('as-animation-controls-widget', 'hour', { variableName: 'myVariable' });
```

Comment:
We're not setting the animation expression to any variable named `myVariable`, right?

Reply:
Nope! it's not needed in this example, good catch!